packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -512,6 +513,9 @@ export async function importBlock(
     );
   }
 
+  // Await writeBlockInputToDb if it was started above
+  await writeBlockInputPromise;
```

Comment:
could you move this to after "Imported block" log but before "Block processed" log
that's give us more detail about timing